examples/archive.html

```diff
@@ -104,17 +111,20 @@ <h3 id="offcanvasRightLabel">Images</h3>
                 let imageRow = document.createElement('div');
                 let image = new Image(150, 150);
                 let placeButton = document.createElement('a');
-
+                let imageArray=[];
                 placeButton.classList.add('btn', 'btn-sm', 'btn-outline-secondary', 'place-button');
                 placeButton.innerHTML = 'Place on map';
-
+                let imageGroup = L.distortableCollection().addTo(map)
```

Comment:
Here, I'm not sure if we have to add it to the map - i think images will be added to the map when the user clicks "Place", right? Maybe we leave out `addTo(map)`?

Reply:
I’ve also made this change